tabular/src/autogluon/tabular/models/rf/rf_model.py

```diff
@@ -35,9 +35,8 @@ def _get_model_type(self):
         if self.params_aux.get('use_daal', False):
             # Disabled by default because it appears to degrade performance
             try:
-                # TODO: Use sklearnex instead once a suitable toggle option is provided that won't impact future models
                 # FIXME: DAAL OOB score is broken, returns biased predictions. Without this optimization, can't compute Efficient OOF.
```

Comment:
OOB score was fixed in scikit-learn-intelex=2021.5 version

Reply:
Thanks for mentioning, this will be very helpful and will enable sklearnex variant of RF to be enabled by default. We are planning 0.4 release in Jan 2022, so will make sure to update sklearnex version if it is released by then.

Reply:
Thanks! This is very helpful. I added a new in-line comment to try the new version once it is released.

Reply:
Hi @agorshk , I tested 2021.5 and it is not yet fixed (train-time oob_score=True works, but not post-fit OOB).

Refer to https://github.com/intel/scikit-learn-intelex/issues/933 for more info

Reply:
> Hi @agorshk , I tested 2021.5 and it is not yet fixed (train-time oob_score=True works, but not post-fit OOB).
> 
> Refer to [intel/scikit-learn-intelex#933](https://github.com/intel/scikit-learn-intelex/issues/933) for more info

Hi @Innixma, thanks for report and reproducer, we'll have a look on this problem.